packages/svelte/src/compiler/phases/2-analyze/visitors/Attribute.js

```diff
@@ -25,6 +25,13 @@ export function Attribute(node, context) {
 		}
 	}
 
+	// special case loading="lazy"
+	if (node.name === 'loading') {
+		if (parent.type === 'RegularElement' && parent.name === 'img') {
```

Comment:
can't we put this into the above if statement?

Reply:
Isn't more clear like this?

Reply:
Not really, it's just as clear as if it was all in the same if statement to me

Reply:
I'll fix it later since I'm about to start driving

Reply:
@trueadm fixed it if you want to give it a second look

Reply:
You can keep the separate if blocks for the two parts, I just meant the nested ifs :D

Reply:
I'm sorry to be a pain.

Reply:
Oh lol that's why I was surprised. Let me split it again 😁

Reply:
i.e.

```js
// special case
if (node.name === 'value' && parent.type === 'RegularElement' && parent.name === 'option') {
  mark_subtree_dynamic(context.path);
}

// special case loading="lazy"
if (node.name === 'loading' && parent.type === 'RegularElement' && parent.name === 'img') {
  mark_subtree_dynamic(context.path);
}
```

Reply:
👍🏻 did it

Reply:
LGTM